resources/views/account/edit.blade.php

```diff
@@ -85,6 +85,20 @@
                     <span class="help-block">This paragraph will go at the top of your public speaker profile page. You can use it to communicate any message you'd like to conference organizers.</span>
                     {{ Form::textarea('profile_intro', null, ['class' => 'form-control']) }}
                 </div>
+
+                <div class="form-group">
+                    {{ Form::label('location', 'Location', ['class' => 'control-label']) }}
+                    <span class="help-block">Enter the city in which you reside and local conference organizers can find you.</span>
+                    {{ Form::text('location', null, ['class' => 'form-control', 'id' => 'autocomplete']) }}
+                </div>
+
+                <div class="form-group">
+                    {{ Form::hidden('neighborhood', null, ['id' => 'neighborhood', 'readonly' => true]) }}
```

Comment:
This is done automatically when using `Form::model` according to [Laravel Collective documentation](https://laravelcollective.com/docs/5.3/html#form-model-binding). 

I tested this out myself and it works just fine.